front/app/components/PostCardsComponents/ViewButtons.tsx

```diff
@@ -32,14 +32,14 @@ import useLocale from 'hooks/useLocale';
 const Container = styled.div`
   display: flex;
   padding: 4px;
-  background: ${darken(0.06, colors.lightGreyishBlue)};
+  background: ${darken(0.06, colors.grey200)};
```

Comment:
Out of scope for this PR but wanted to mention that we should get rid of these 'derived' colors and just use a color that's in the design system

Reply:
Definitely! Agree it's problematic and I'm trying delete them when I see them (if possible) but they are still pretty heavily used, unfortunately. Will make a note in the docs about using component library colors with their nuances over this polished thing.